docs/content/user-guide/inference.md

```
@@ -8,6 +8,21 @@ This guide explains how to make inference requests to models through the MaaS pl
 
 ---
 
+## Endpoint Overview
+
+MaaS provides an **OpenAI-compatible** inference endpoint. Send all requests to a single gateway URL and specify the model in the request body:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)